client/app/assets/less/inc/dropdown.less

```diff
@@ -29,10 +29,12 @@
 
     &:not([class*="bg-"]) {
         & > li > a {
-            color: #4C4C4C;
```

Comment:
@tonyjiangh I think, [this is a better place](https://github.com/getredash/redash/blob/master/client/app/assets/less/inc/dropdown.less#L9-L15) for this CSS. @kocsmy - WDYT?

Reply:
Yes, we have a disabled class for li already. We'll need to adjust the color a bit more gray to indicate it's disabled.